README.md

```diff
@@ -0,0 +1,35 @@
+# Unimus Backup Extractor
+
+This repository contains the Unimus Backup Extractor.  
+This extractor is used to create local copies of unimus device backups. 
+
+### TL;DR
+Requirements: `bash`, `jq`, `base64`.
+```text
```

Comment:
should show user how to get functional - which requires 2 steps, config setup, running script